node/src/main/scala/coop/rchain/node/web/WebApiRoutes.scala

```diff
@@ -36,7 +36,11 @@ object WebApiRoutes {
             case err: BlockApiException => BadRequest(err.getMessageSafe.asJson)
             case err: Throwable         =>
               // Logging only unanticipated errors, not related to Block API or input parsing (user errors)
-              Log[F].error("HTTP API response error", err) *> BadRequest(err.getMessageSafe.asJson)
+              // To indicate that error is not related to internal node error and
+              // node can continue running, error logged as a warning.
+              Log[F].warn("HTTP API response error") *>
+                Sync[F].delay(err.printStackTrace()) *>
```

Comment:
Please add overload to `warn` method accepting Throwable similar how it's defined for `error` method.
Also _cause_ should be defined as a thunk the same as the message.
https://github.com/rchain/rchain/blob/76f14c516bac3d6be2f91318c9b202b4239465bb/shared/src/main/scala/coop/rchain/shared/Log.scala#L38-L46